.github/workflows/update-links.yml

```
@@ -9,6 +9,10 @@ on:
   schedule:
     - cron: "0 0 * * *"
 
+concurrency:
+  group: update-redirect-links
+  cancel-in-progress: false
```

I think it should cancel in progress to start the most recent one, no?

---

```suggestion
  # Allow existing runs to complete to avoid branch collisions
  cancel-in-progress: false
```
Worth adding this to clarify?